settings.gradle

```diff
@@ -39,7 +39,8 @@ include ":opentelemetry-all",
         ":opentelemetry-exporter-jaeger",
         ":opentelemetry-exporter-jaeger-thrift",
         ":opentelemetry-exporter-logging",
-        ":opentelemetry-exporter-otlp",
```

Comment:
do we envision eventually having a module that ties the two (or maybe 3 with logs) together? Should we keep that module now for the 1.0 release so that people don't have to adjust their dependencies?